emg/material/teaching/mod01.en.md

```diff
@@ -7,7 +7,7 @@ ref: teaching-mod01
 ---
 
 <div class="sixteen_nine_outer">
-    <iframe src="https://onedrive.live.com/embed?resid=E964CF1763131888%21936&authkey=!AGish6pueWC33KI&em=2" width="100%" height="100%" class="aspect_ratio_inner" frameborder="0" scrolling="no"></iframe>
+    <iframe src="https://onedrive.live.com/embed?resid=E964CF1763131888%21958&amp;authkey=!AKNKWKBmCbPkypk&amp;em=2&amp;wdAr=1.7777777777777777" width="100%" height="100%" class="aspect_ratio_inner" frameborder="0" scrolling="no"></iframe>
```

Comment:
Is the conversion of `&` into `&amp;` intended?
Do you know what the added parameters `em` e `wdAr` do? (For my own understanding in case these were intended and might need to be applied elsewhere too)